packages/manager/modules/pci/src/projects/project/serving/namespace/models/add/add.constants.js

```diff
@@ -1,7 +1,7 @@
 export const PRESET_IMAGE = 'preset-image';
 export const BUILD_IMAGE = 'build-image';
-export const EXTENSION = ['.h5', '.onnx', '.pmml'];
+export const EXTENSION = ['.h5', '.onnx', '.pmml', '.pt', '.pth', '.pkl'];
 export const FOLDER_MODE = 'folder';
 export const FILE_MODE = 'file';
 export const COMPATIBILITY_GUIDE_URL =
-  'https://docs.ovh.com/gb/en/serving-engine/compatibility-matrix';
+  'https://docs.ovh.com/gb/en/ml-serving/compatibility-matrix';
```

Comment:
what about fr and other language guides? its good to fetch guide URL based on language even though guides are not available in all languages. When available it will work without any changes. Default it to en or fr.